tv_apps/smart-system/js/browser_context_menu.js

```diff
@@ -299,11 +299,10 @@
 
   BrowserContextMenu.prototype._listAddAppToAppsItem = function(manifestURL) {
     return new Promise((resolve, reject) => {
-      var _ = navigator.mozL10n.get;
       if (!AppInstallManager.getAppAddedState(manifestURL)) {
         resolve({
           type: BUTTON_TYPE,
-          textRaw: _('add-to-apps'),
+          textL10nId: 'add-to-apps',
```

Comment:
![2016-02-24 6 41 47](https://cloud.githubusercontent.com/assets/2406290/13282954/753fabf2-db26-11e5-94b4-bc40b9e1c89f.png)
I found an icon disappeared in context menu (picture from "WebApps" app)
Seems there's a child `<div>` that contain an icon (see line 218) and it's overwritten by `l10n-id` filling process. Should we modify the dom structure a little for this case?